apollo-router/src/services/mod.rs

```diff
@@ -302,28 +302,45 @@ impl QueryPlannerRequest {
     pub fn new(
         query: String,
         operation_name: Option<String>,
-        query_plan_options: QueryPlanOptions,
         context: Context,
     ) -> QueryPlannerRequest {
         Self {
             query,
             operation_name,
-            query_plan_options,
+            query_plan_options: QueryPlanOptions::default(),
```

Comment:
Should `QueryPlannerRequest::new` take `&Configuration` in order to have the correct value here?

Reply:
For now it's ok but if we need this, it's a bigger topic because we need to provide access to global configuration to plugins